# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1670880537 -3600
# Parent  61b753b965abb077ce3310cda9681d82e24a369d
Misc: Use Betterbird's own donation appeal URL.

diff --git a/mail/app/profile/all-thunderbird.js b/mail/app/profile/all-thunderbird.js
--- a/mail/app/profile/all-thunderbird.js
+++ b/mail/app/profile/all-thunderbird.js
@@ -1324,10 +1324,10 @@ pref("print.print_headercenter", "&T");
 pref("print.print_headerright", "");
 
 // Enable Masonry Layout for AddressBook.
 pref("layout.css.grid-template-masonry-value.enabled", true);
 
 // End of year donation appeal.
 pref("app.donation.eoy.version", 1);
 pref("app.donation.eoy.version.viewed", 0);
-pref("app.donation.eoy.url", "https://www.thunderbird.net/thunderbird/102.0/eoy/");
+pref("app.donation.eoy.url", "https://www.betterbird.eu/end-of-year/");
 
